src/test/java/core/basesyntax/WorkWithFileTests.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Operations;
+import core.basesyntax.service.FileReader;
+import core.basesyntax.service.FileWriter;
+import core.basesyntax.service.impl.ReadFromCsvFileImpl;
+import core.basesyntax.service.impl.WriteToCsvFileImpl;
+import core.basesyntax.strategy.BalanceHandler;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import core.basesyntax.strategy.PurchaseHandler;
+import core.basesyntax.strategy.ReturnHandler;
+import core.basesyntax.strategy.SupplyHandler;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class WorkWithFileTests {
+
```

Comment:
let's test each service in the separate test class

please test the services independently of each other - if you test fruitStorageService, you should not read from file in this test class

remember that we should static storage everywhere without creating a field - you can also set the values to it directly, test some behaviour and then clear the storage